markdown/3.x/en/ff-navigation.md

```diff
@@ -261,6 +261,8 @@ The Element with the 'data-search' property will only be visible on elements whi
 ## Custom navigation data
 
 To use custom navigation data instead of data retrieved from FACT-Finder, set `subscribe="false"` and assign an array of navigation items to the `navigationData` property.
+You can also subscribe to the default topic and inspect data passed as an argument to `resultDispatcher.subscribe`. function. For more details how subscribe to topic, please visit
```

Comment:
Putting the purpose reads a little easier because you can immediately estimate how relevant the sentence is to what you are looking for.
```suggestion
To inspect the structure of such data you can subscribe to the default topic through `ResultDispatcher.subscribe` and read its argument. For more details on how to subscribe to topics, please visit the
```

Reply:
changed